sympy/functions/elementary/complexes.py

```diff
@@ -466,7 +466,7 @@ def eval(cls, arg):
                     if base.func is cls and exponent is S.NegativeOne:
                         return arg
                     return Abs(base)**exponent
-                if base.is_positive == True:
+                if base.is_positive is True:
```

Comment:
Either way, you should never `== True`. I'll fix it so that it will avoid the `is True` as well.